### DISCUSSION TITLE
Fork interface

### DISCUSSION THREAD

**Discussion by NhanAZ** [community/community#14936] [Other Feature Feedback, Questions, & Ideas]
With the old interface I easily fork a repo with 2 clicks
In the new interface, the number of clicks to fork a repo is at least 3

---

Not to mention I can't fork a repo that's already fork in the new interface
I think Github should allow users to choose between the two instead of forcing them to use an inconvenient interface!

**Comment by vincenttjia** [comment]
Yeah, please give us a switch in settings to choose between the new or old interface.

Or at least give us a quick fork/go to your fork option in the Command Palette.

**Comment by NhanAZ** [comment]
The new fork interface is only useful for people who don't use fork often
***
Those who often use it, obviously, this is a huge inconvenience...

**Comment by Jacke** [comment]
Please give us a possibility to choose the fork interface!

**Comment by unitycoder** [comment]
github, is this getting fixed soon?

**Reply by NhanAZ** [reply]
I don't think they would be interested in a discussion with few people discussing...
Anyway, hopefully this issue will be fixed soon

**Comment by slonopotamus** [comment]
What's worse, I no longer see an easy way to navigate from parent repo to my own fork of it. With old UI, I used "Fork" button and got a popup where I could click on my fork.

**Reply by darthpaul-sanuk** [reply]
This is the main issue for me with the new design.

**Reply by patel-zeel** [reply]
Yes, this is the most painful part of this new change.

**Reply by atsju** [reply]
well you can still click on fork but then you need to search through the 350+ other forks to find yours

**Reply by patel-zeel** [reply]
As of now, I am using this hack: 1) go to the URL 2) replace the original organization name with your username.

**Comment by wutthichai46** [comment]
#12341

**Reply by atsju** [reply]
This is absolutely not linked to the question. Why did you post ?

**Comment by vincenttjia** [comment]
Hi Everyone I think this works for me. check out the updated fork interface

<img width="763" alt="image" src="https://user-images.githubusercontent.com/48272053/166863191-537b3a24-48f6-41ed-b869-d7d62768c2f6.png">

**Reply by NhanAZ** [reply]
I know this, but it's still inconvenient for me!

**Reply by NhanAZ** [reply]
Take a look at this: https://github.com/pmmp/PocketMine-MP/network/members
Took time to load and check my fork

**Reply by vincenttjia** [reply]
Check under these forks already exist(it only shows fork owned by me), and It's as good as the old interface only that it take you to a diffrent page.

Btw the screenshot isn't taken after clicking the view all existing forks

**Reply by vincenttjia** [reply]
Btw I'm not sure though. Am I the only one that have the new updated interface and that it is rolled out over time?

**Reply by atsju** [reply]
Interfaces are not rolled out over time.
Have a look here and at following comments https://github.com/github/feedback/discussions/15418#discussioncomment-2665029
The interface is not OK when you have an multiple organisation (for example your the default one with your user name plus an organisation you create for a project)
